applications/API/serializers.py

```
@@ -0,0 +1,487 @@
+    status = serializers.CharField(source="get_status_display", read_only=True)
+    nsfw = serializers.BooleanField()
+    id = serializers.ReadOnlyField()
+    added_by = serializers.SerializerMethodField()
+
+    @extend_schema_field(serializers.CharField())
+    def get_added_by(self, obj):
+        """Format the added_by field to protect user privacy."""
+        try:
+            if not obj.added_by:
+                return None
+            elif obj.added_by.first_name and not obj.added_by.last_name:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)